map_block_msg[] blocks
